File: Unidad 4/p3_u4.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            double f=0 , k = 0;
            int temperaturaT;
            Console.WriteLine("Ingrese las temperaturas: ");
            temperaturaT = Convert.ToInt32(Console.ReadLine());

            double[] temperaturaC = new double[temperaturaT];

            for (int i = 0; i < temperaturaT; i++)
            {
                Console.WriteLine("Ingrese la temperatura en Celsius" + i +":");
                temperaturaC[i] = Convert.ToDouble(Console.ReadLine());

                f = temperaturaC[i] * (9 / 5) + 32;
                Console.WriteLine("Temperatura en Farenheit: " + f);

                k = temperaturaC[i] + 273.15;
                Console.WriteLine("Temperatura en Kelvin: "+k);

            }
           

               
        }
    }
}

File: Unidad 4/P3_u4.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p3_u4;
import java.util.Scanner;
/**
 *
 * @author richy
 */
public class P3_u4 {

    public static void main(String[] args) {
       Scanner sc = new Scanner(System.in);
        double f = 0, k= 0;
         int temperaturaT;

            System.out.println("Ingrese la cantidad de temperaturas a procesar: ");
            temperaturaT = sc.nextInt();

            double[] temperaturaC = new double[temperaturaT];
            for (int i = 0; i < temperaturaT; i++)
            {
                System.out.println("Ingrese la temperatura en Celsius: ");
                temperaturaC[i] = sc.nextInt();

                f = temperaturaC[i] * (9 / 5) + 32;
                System.out.println("Temperatura en Farenheit: " + f);

                k = temperaturaC[i] + 273.15;
                System.out.println("Temperatura en Kelvin: " + k);
            }

        }
    }
